keadm/cmd/keadm/app/cmd/cloud/init.go

```diff
@@ -97,6 +97,9 @@ func addJoinOtherFlags(cmd *cobra.Command, initOpts *types.InitOptions) {
 
 	cmd.Flags().StringVar(&initOpts.AdvertiseAddress, types.AdvertiseAddress, initOpts.AdvertiseAddress,
 		"Use this key to set SANs in certificate of cloudcore. eg: 10.10.102.78,10.10.102.79")
+
+	cmd.Flags().StringVar(&initOpts.DNS, types.DomainName, initOpts.DNS,
+		"Use this key to set SANs in certificate of cloudcore. eg: www.cloudcore.cn,www.kubeedge.cn")
```

Comment:
```suggestion
		"Use this key to set domain names in cloudcore's certificate SubAltNames field. eg: www.cloudcore.cn,www.kubeedge.cn")
```

Also update the ip field above.

Reply:
@fisherxu  ok